ndmspc.sh

```diff
@@ -19,12 +18,16 @@ if [[ $ALIBUILD_NDMSPC_TESTS ]]; then
   # Impose extra errors.
   CXXFLAGS="${CXXFLAGS} -Werror -Wno-error=deprecated-declarations"
 fi
-cmake --version
 
 # When O2 is built against Gandiva (from Arrow), then we need to use
 # -DLLVM_ROOT=$CLANG_ROOT, since O2's CMake calls into Gandiva's
 # -CMake, which requires it.
-cmake "$SOURCEDIR" "-DCMAKE_INSTALL_PREFIX=$INSTALLROOT"
+cmake "$SOURCEDIR" "-DCMAKE_INSTALL_PREFIX=$INSTALLROOT"          \
+      -G Ninja                                                    \
```

Comment:
You probably need a build_requires for ninja.

Reply:
Done